chart/values.yaml

```diff
@@ -82,18 +89,21 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
+# Provided by Leonardo
 nodeSelector: {}
   # cloud.google.com/gke-nodepool: "nodepool-name"
 
 tolerations: []
 
 affinity: {}
 
+# either GCE PD provided by Leonardo; or hostPath provided by terra-docker-local.sh
 persistence:
   enabled: true 
   storageClass: "manual"
-  # size: 50G
-  # accessMode: ReadOnlyMany 
+  size: 10G
```

Comment:
should this be configurable?

Reply:
Yes - values in here are essentially defaults, I think Leo would need to pass this as an arg depending on the disk size